test/cli/compiler/test-qx-package.js

```diff
@@ -2,37 +2,37 @@ const test = require("tape");
 const fs = require("fs");
 const testUtils = require("../../../bin/tools/utils");
 const fsPromises = testUtils.fsPromises;
-
-test("Create app", async assert => {
+require("process").chdir(__dirname);
+test("Create app", async t => {

```

Comment:
renaming of `assert`

Reply:
As before, I think we should switch to the tape convention so that others aren't confused.